feeadjuster/feeadjuster.py

```diff
@@ -51,7 +68,28 @@ def maybe_setchannelfee(plugin: Plugin, scid: str, base: int, ppm: int):
         plugin.rpc.setchannelfee(scid, base, ppm)
         return True
     except RpcError as e:
-        plugin.log("Could not adjust fees for channel {}: '{}'".format(scid, e), level="warn")
+        plugin.log(f"Could not adjust fees for channel {scid}: '{e}'", level="warn")
+    return False
+
+
+def significant_update(plugin: Plugin, scid: str):
+    channel = plugin.adj_balances[scid]
+    last_liquidity = channel.get("last_liquidity")
+    if last_liquidity is None:
+        return True
+    # Only update on substantial balance moves to avoid flooding, and add
+    # some pseudo-randomness to avoid too easy channel balance probing
+    update_threshold = plugin.update_threshold
+    update_threshold_abs = int(plugin.update_threshold_abs)
+    if not plugin.deactivate_fuzz:
+        r = random.uniform(-0.015, 0.015)
+        update_threshold += r
+        update_threshold_abs += update_threshold_abs * r
```

Comment:
Suggestion: `r` should be taken twice: once for fuzzing `update_threshold` and another one for `update_threshold_abs`. In this case I would skip the local asignment of `r` altogether and simply do: `update_threshold += random.uniform(-0.015, 0.015)` and `update_threshold_abs += random.uniform(-0.015, 0.015)`